Attribution.txt

```diff
@@ -178,3 +178,18 @@ https://github.com/leodido/go-urn
 
 github.com/lib/pq (MIT) https://github.com/lib/pq
 https://github.com/lib/pq/blob/master/LICENSE.md
+
+github.com/fatih/color (MIT) https://github.com/fatih/color
+https://github.com/fatih/color/blob/master/LICENSE.md
+
+github.com/hashicorp/go-hclog (MIT) https://github.com/hashicorp/go-hclog
```

Comment:
https://github.com/hashicorp/go-hclog isn't in our approved list. But since it is `hashicorp`, I assume we can just add it?

Reply:
Yes - this will need TSC approval.  Suggesting we approve with "pending TSC approval", then pull it back out and revisit Consul 1.8.1 if not approved.  This in order to get builds going again.

Reply:
now approved